kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -483,7 +619,12 @@
         fetchMoreResources,
         resetWorkingResourcePool,
         contentPresentInWorkingResourcePool,
-        //contentList,
+        questionCount,
```

Comment:
`contentPresentInWorkingResourcePool` (named on the line above) is failing for me when I go through to the QA Channel -> Exercises

I have a KA Perseus topic which has no children so when the card for that KA Perseus topic is passed to `contentPresentInWRP` it has no children, which chokes up the function.

Reply:
Might just work well to filter out empty topics to begin with - which I thought we'd been doing 🤔